scripts/prettier.js

```diff
@@ -12,6 +12,8 @@ const yargs = require('yargs');
 const { LANGUAGES } = require('docs/src/modules/constants');
 const listChangedFiles = require('./listChangedFiles');
 
+const workspaceRoot = path.resolve(__dirname, '../');
```

Comment:
For #22071
```suggestion
const workspaceRoot = process.cwd();
```

Reply:
that's not the workspace root and wrong.

Reply:
If you need to run it in a different repo fork the script. We can't maintain scripts that work in arbitrary repositories.

Reply:
We don't have the bandwidth to maintain two different scripts. We need to share the logic, this is a hard requirement. Is there a way we could provide a configuration variable in this case? It could be an env or a [args](https://github.com/mui-org/material-ui/pull/22071/files#diff-95920dbc1fc78b4318782fbc23d556948bb0cb19e96c9400933e820051b36c41R121).

Reply:
> We don't have the bandwidth to maintain two different scripts solving the same problem.

We're not solving the same problem so this doesn't apply. Feel free to abstract it into a generic script. For now we're solving problems in this repo.

Reply:
I don't understand. Both repositories are trying to format the sources efficiently with prettier. How is it different? I want to emphazie it. material-ui-x is a different folder of the same codebase.